app/assets/javascripts/components/common/editable_input.es6.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * @prop label          - label of field to show
+ * @prop name           - name of input component
+ * @prop data           - current input for label
+ * @prop editable       - true if fields are editable
+ * @prop handleChange   - callback function when form inputs change
+ * @prop specialHandler - callback for special input types
+ */
+
+class EditableInput extends React.Component {
+
+  renderOptions(type) {
+    var optionsArray = []
+    var retOptions = []
+    var i = 0;
+    var j = 0;
+    switch(type) {
+      case 'gender':
+        optionsArray = GENDERS;
+        break;
+      case 'income_range':
+        optionsArray = INCOME_RANGES;
+        break;
+      case 'household_number':
+        optionsArray = HOUSEHOLD_NUMBER;
+        j = 1;
```

Comment:
we need to add the case for 'state' since it is a dropdown menu too (look at student_form for an example)

- 'school_level'
- 'travel_distance' (this needs to be added on the field)

go to the student form to make sure the fields that should be editable are editable

We need to add changing password (but that can happen later)